modules/ROOT/pages/domain-based-application-cookbook.adoc

```diff
@@ -0,0 +1,154 @@
+= Test Domain-Based Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MUnit 2.x allows you to write MUnit test suites for your Mule applications that share their resources with a Mule domain without the need for any extra configurations.
+
+== Base Mule Application
+
+Consider a Mule domain called `domain-config` that defines a global configuration for the HTTP listeners of all the Mule applications under it:
+
+[.ex]
+=====
+[discrete.view]
+===== Studio Visual Editor
```

Comment:
I'm not sure if it is clearer to simply provide these internal headings with their examples, or to make these introductory sentences to the examples, such as "The Anypoint Studio Visual Editor configuration looks like this:"
Just something to think about, for this ===== and the others below.